### PR TITLE
[WIP] Add Unicode Codepoints to Emoji Map

### LINKED ISSUE #8: Add unicode codepoint options for all emojiwords
From #5, add Unicode codepoint words for all the current entries.

https://github.com/vbatts/emojisum/blob/master/emoji/emojimap.json
Now supports having multiple options per position in the list, provided they render the same emoji.
Initially we have the colon notation, but would like the unicode codepoint option as well.

### PR DESCRIPTION
**Pull request by smunilla**
Reference: #8

### CODE REVIEW

**Comment by vbatts** [comment]
updated the comment because it doesn't completely fix #8

File: emoji/emojimap.json [hunk 1]
```diff
@@ -2,38 +2,38 @@
...
-    [":blue_car:"],
-    [":boar:"],
-    [":bomb:"],
-    [":boom:"],
-    [":bow:"],
-    [":boy:"],
-    [":broken_heart:"],
-    [":bulb:"],
-    [":bus:"],
+    [":+1:", "U+1F44D"],
```
Comment (smunilla) [inline emoji/emojimap.json]: I went with Thumbs-up for this one

File: emoji/emojimap.json [hunk 2]
```diff
@@ -2,38 +2,38 @@
...
+    [":banana:", "U+1F34C"],
+    [":barber:", "U+1F488"],
+    [":bathtub:", "U+1F6C1"],
+    [":beer:", "U+1F37A"],
+    [":bell:", "U+1F514"],
+    [":bicyclist:", "U+1F6B4"],
+    [":bird:", "U+1F426"],
+    [":birthday:", "U+1F382"],
+    [":blossom:", "U+1F33C"],
+    [":blue_car:", "U+1F693"],
```
Comment (smunilla) [inline emoji/emojimap.json]: Unicode does not have cars by color, so I thought Police Car was the closest choice.

File: emoji/emojimap.json (hunk 1 above)
Comment (vbatts) [inline emoji/emojimap.json]: let's switch it to `:thumbsup:` (:thumbsup:) then

**Pull request merged**